src/controllers/userController.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response, NextFunction } from "express";
+import { getRepository } from "typeorm";
+import { conn } from "../Utils/dbConnection";
+import { User } from "../entity/user";
+import prompt from "../Utils/apiResponses";
```

Comment:
If you were exporting the methods directly and not using a class, you would do this
```suggestion
import {successWithPayload, errorMessage} from "../Utils/apiResponses";
```